servo-media/src/audio/macros.rs

```diff
@@ -4,6 +4,9 @@ macro_rules! make_message_handler(
         fn message(&mut self, msg: ::audio::node::AudioNodeMessage, sample_rate: f32) {
             match msg {
                 ::audio::node::AudioNodeMessage::$node(m) => self.handle_message(m, sample_rate),
+                ::audio::node::AudioNodeMessage::GetInputCount(tx) => tx.send(self.input_count()).unwrap(),
```

Comment:
We don't need these; the DOM will maintain these values itself I think.

All of the invariants will be upheld by the DOM side as well (aside from invariants that require maintaining an audio param timeline or a graph to validate)